src/core/referencingfeaturelistmodel.h

```diff
@@ -55,6 +57,18 @@ class ReferencingFeatureListModel : public QAbstractItemModel
     int columnCount( const QModelIndex &parent = QModelIndex() ) const override;
     QVariant data( const QModelIndex &index, int role = Qt::DisplayRole ) const override;
 
+    //! Returns the relation id connecting the parent feature with the children in this model
```

Comment:
```suggestion
    //! Returns the id of the relation connecting the parent feature with the children in this model
```
Sorry :-)

Reply:
I spent 2 minutes trying to figure out how to write it like proposal. Seems English is switched off this morning.

Reply:
Well 2 minutes is not that bad...